USAGE.md

```diff
@@ -1,9 +1,10 @@
 # Usage
 
-We support three container images currently:
+We publish four container images currently:
 - [Tornjak Backend](https://github.com/spiffe/tornjak/pkgs/container/tornjak-be): This image can be deployed as a sidecar with any SPIRE server. 
 - [Tornjak Manager](https://github.com/spiffe/tornjak/pkgs/container/tornjak-manager): A container that runs this image exposes a port to register multiple Tornjak backends and forward typical commands to multiple Tornjak backends from one API. 
 - [Tornjak Frontend](https://github.com/spiffe/tornjak/pkgs/container/tornjak-fe): This image is typically deployed after the Tornjak Backend or Manager are deployed, as it requires a URL to connect directly to the Tornjak backend API.  
+- [Tornjak](https://github.com/spiffe/tornjak/pkgs/container/tornjak): This image can deployed as one sidecare container alongside SPIRE that also includes the Tornjak frontend component. NOTE: this feature is experimental currently, but useful for quick deployment. 
```

Comment:
Maybe we could rephrase this as:
"This image containing both Tornjak Backend and Frontend components can be deployed as a sidecar alongside SPIRE Server container"

And since this would be our new default, we can remove the NOTE, unless you think we should provide some kind of the warning?